README.md

```diff
@@ -18,6 +18,8 @@ To get started with RIBs, please refer to the [RIBs documentation](https://githu
 
 To get more hands on with RIBs, we have written a [series of tutorials](https://github.com/uber/RIBs/wiki) that run you through the main aspects of the architecture with hands-on examples.
 
+To read about the backstory on why we created RIBs, see [this blog post](https://eng.uber.com/new-rider-app/) we wrote when releasing RIBs in production the first time and see [this short video](https://www.youtube.com/watch?v=Q5cTT0M0YXg) where we discussed how the RIBs architecture works.
```

Comment:
I can't remember. Did yi's talk make sense without the tony's scoping talk coming first?

Reply:
Yeah I would say it makes sense - I watched it over the weekend, in saying that though I have a pretty good understanding of RIB's ;-)